src/osbuild-manifests/coreos.osbuild.s390x.secex.mpp.yaml

```diff
@@ -0,0 +1,385 @@
+              # because there is a symlink that is created in the root of the boot
+              # filesystem by OSTree (boot -> .) that makes it so that /boot paths
+              # will always work.
+              bootprefix: true
+      # Mountpoint for filesytem labeled `se`, where `sdboot` image gets stored
+      - type: org.osbuild.mkdir
+        options:
+          paths:
+            - path: /se
+      - type: org.osbuild.ignition
+      # Deploy via OSTree repo if specified, otherwise ociarchive or container.
+      - mpp-if: ostree_repo != ''
+        then:
+          type: org.osbuild.ostree.deploy
+          options:
+            osname:
+              mpp-format-string: '{osname}'
+            remote: fedora
+            mounts:
+              - /boot
+            kernel_opts:
+              - rw
+             ## '$ignition_firstboot' only works with GRUB, not available on s390x
+             #- '$ignition_firstboot'
+              - mpp-format-string: '{extra_kargs}'
+          inputs:
+            commits:
+              type: org.osbuild.ostree
+              origin: org.osbuild.source
+              mpp-resolve-ostree-commits:
+                commits:
+                  - ref: $ostree_ref
+                    remote:
+                      url: $ostree_repo
+        else:
+          mpp-if: ociarchive != ''
+          then:
+            type: org.osbuild.ostree.deploy.container
+            options:
+              osname:
+                mpp-format-string: '{osname}'
+              target_imgref:
+                mpp-format-string: '{container_imgref}'
+              mounts:
+                - /boot
+              kernel_opts:
+                - rw
+               ## '$ignition_firstboot' only works with GRUB, not available on s390x
+               #- '$ignition_firstboot'
+                - mpp-format-string: '{extra_kargs}'
+            inputs:
+              images:
+                type: org.osbuild.containers
+                origin: org.osbuild.pipeline
+                references:
+                  name:oci-archive:
+                    name: coreos.ociarchive
+          else:
+            type: org.osbuild.ostree.deploy.container
+            options:
+              osname:
+                mpp-format-string: '{osname}'
+              target_imgref:
+                mpp-format-string: '{container_imgref}'
+              mounts:
+                - /boot
+              kernel_opts:
+                - rw
+               ## '$ignition_firstboot' only works with GRUB, not available on s390x
+               #- '$ignition_firstboot'
+                - mpp-format-string: '{extra_kargs}'
+            inputs:
+            images:
+              type: org.osbuild.containers
+              origin: org.osbuild.source
+              mpp-resolve-images:
+                images:
+                  - source: $container_repo
+                    tag: $container_tag
```

Comment:
Needs indentation. This was fixed recently-ish in the non-secex file, so might be worth rebasing your changes on top of the latest version.

This is a good argument actually for trying to merge the two and using mpp conditionals. Were you able to try it out?

Reply:
Fixed indentation. 

>Were you able to try it out?

Not yet